game_api.txt

```diff
@@ -690,6 +690,7 @@ Creates panes that automatically connect to each other
 		groups = {group = rating}, -- Uses the known node groups, see [Known damage and digging time defining groups]
 		sounds = SoundSpec,        -- See [#Default sounds]
 		recipe = {{"","","","","","","","",""}}, -- Recipe field only
+		use_texture_alpha = true, -- For colored glass panes
```

Comment:
Please mention this is an optional parameter that defaults to false.
+1 once that's done.